package.json

```diff
@@ -92,7 +92,7 @@
     "@types/react-redux": "7.1.7",
     "@types/react-router": "5.1.4",
     "@types/react-router-dom": "5.1.3",
-    "@types/uuid": "3.4.8",
+    "@types/uuid": "7.0.0",
```

Comment:
I'm wondering why this is working now, and last week it didn't 🤔

Reply:
Not sure either. I spend too much time looking into dependency issues the last weeks. I am just happy if something works :)